server/views/manage/premises/show.njk

```diff
@@ -42,19 +44,60 @@
         PremisesUtils.summaryListForPremises(premises)
     ) }}
 
-    {% if placements %}
-        {{ mojSubNavigation({
-            label: 'Sub navigation',
-            items: PremisesUtils.premisesTabItems(premises, activeTab)
-        }) }}
+    {% if showPlacements %}
+        <section>
+            <h2 class="govuk-heading-m">All bookings</h2>
```

Comment:
This follows the design, but the title makes no sense, in any tab actually but particularly on the search tab as it's clearly not showing 'All bookings'
Maybe have a chat with Stevie. Could be just a content change.

Reply:
We'll need to audit the messaging consistency it seems, as there are both references to 'Bookings' and 'Placements' around the whole flow. This is probably better tackled separately.